pkgs/development/tools/sourcetrail/default.nix

```diff
@@ -0,0 +1,65 @@
+{ stdenv, fetchurl, patchelf, makeWrapper, expat, openssl, zlib }:
+
+stdenv.mkDerivation rec {
+  name = "sourcetrail-${version}";
+  version = "2018.3.55";
+
+  src = fetchurl {
+    name = "sourtrail.tar.gz";
+    url = "https://www.sourcetrail.com/downloads/${version}/linux/64bit";
+    sha256 = "6f5fbbecc221e7165ecbf1c4d208e302dade4feea9d43eb3265521efc0a376e2";
+  };
+
+  dontBuild = true;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out
+    mv -v setup/share $out
+    mkdir -p $out/share/icons
+    mv -v data/gui/icon/logo_1024_1024.png $out/share/icons/sourcetrail.png
+    mv -v data/gui/icon/project_256_256.png $out/share/icons/project-sourcetrail.png
+
+    mkdir -p $out/share/sourcetrail/doc
+    mv -v README EULA.txt $out/share/sourcetrail/doc
+    mv -v plugin $out/share/sourcetrail
+
+    mkdir -p $out/opt
+    mv -v * $out/opt
+
+    mkdir -p $out/bin
+    patchShebangs $out/opt/Sourcetrail.sh
+    makeWrapper $out/opt/Sourcetrail.sh $out/bin/sourcetrail \
+      --prefix LD_LIBRARY_PATH : "${ldPath}"
+    patchelf \
+      --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" \
+      --set-rpath "$out/opt/lib:${rpath}" \
+      $out/opt/Sourcetrail
+
+    for l in $out/opt/lib/*.so.* $out/opt/lib/platforms/*.so.*; do
+      patchelf --set-rpath "$out/opt/lib:${rpath}" "$l"
+    done
+
+    substituteInPlace \
+      $out/share/applications/sourcetrail.desktop \
+      --replace /usr/bin/ $out/bin/
+
+    # FIXME: not working
```

Comment:
I can't seem to change the permission of default configuration files that are copied to the user's home directory upon first launch. Does anyone have a clue why this is happening?

Reply:
Everything in the nix store is always read only.

Reply:
@symphorien Do you mean even during `installPhase` ?